packages/main/src/MultiComboBox.ts

```diff
@@ -1422,6 +1433,7 @@ class MultiComboBox extends UI5Element {
 	}
 
 	onBeforeRendering() {
+		// this.toggleAttribute("open", this.open);
```

Comment:
I tried to introduce this additional code, but the behavior of the component does not change. Please make additional checks if it is needed at all.